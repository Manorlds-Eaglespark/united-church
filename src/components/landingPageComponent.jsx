import React from 'react';
import { Link } from 'react-router-dom';
import logoImage from '../assets/images/login_image.png';
import loginImage from '../assets/images/google.png';

const landingPageComponent = () => (
  <div>
    <div className="pageHeading">
      <span className="logo-r">United Church</span>
    </div>

    <div>
    <img className="center landing_image" src={logoImage} alt="" />
    <p className="logo-text color-blue">Sharing across a united CHURCH </p>
    <Link to="/select">
      <img className="center google-login" src={loginImage} alt="" />
    </Link>
  </div>
  </div>
);

export default landingPageComponent;
