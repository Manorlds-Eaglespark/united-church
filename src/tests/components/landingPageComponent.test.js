import React from 'react';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';
import landingPageComponent from '../../components/landingPageComponent';


describe('<landingPageComponent />', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<landingPageComponent />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
