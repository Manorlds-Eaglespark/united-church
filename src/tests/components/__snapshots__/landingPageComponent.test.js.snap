// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<landingPageComponent /> should render correctly 1`] = `<landingPageComponent />`;
